Formula/s/snakeviz.rb

```diff
@@ -1,3 +1,4 @@
+require "cgi"
```

Comment:
Can we lazy load this by moving this to the start of the `test` block instead?

Reply:
Yes, that should be possible.